dom.go

```diff
@@ -299,6 +299,14 @@ func (h *HTML) populateClassNameProperty() {
 	}
 }
 
+func (h *HTML) Add(m ...MarkupOrComponentOrHTML) *HTML {
+	for _, m := range m {
+		apply(m, h)
+	}
+
+	return h
+}
```

Comment:
It's not clear to me why this is needed or desired? What problem does this solve? Please create an issue and I can fix it or send this as a separate PR and we can discuss there.